### DISCUSSION TITLE
Desktop Shortcut for Saved Session

### DISCUSSION THREAD

**Discussion by 1RonRock** [sienori/Tab-Session-Manager#1024] [Ideas]
Would it be possible to have a desktop shortcut to a session? I often have many saved tabs that eventually gets too many for my FFox browser to efficiently handle. So I have to choose the option tor SM to do nothing on startup, then after FF opens find the desired session and open it, close the previous FF session and move on. 

Would be nice if there is a way to have a desktop shortcut to a session so that it can be opened with a simple double click.

**Comment by ninmonkey** [comment]
![image](https://user-images.githubusercontent.com/3892031/173461862-00f4c76c-98a8-4ebd-9274-6192b11ad8d2.png)

That used to be easy with this command, it would

- close all currently open windows
- open multiple windows and tabs from the group

But right now it seems to do nothing, when I click on it. Unless I'm misunderstanding a setting somewhere?
